contracts/ExampleTrustcoin2.sol

```
@@ -31,31 +32,24 @@ contract ExampleTrustcoin2 is ERC20TokenInterface, SafeMath {
   address public migrationMaster;
   address public newTokenAddress;
 
-  event OutgoingMigration(address owner, uint256 value);
-  event IncomingMigration(address owner, uint256 value);
   event MigrationFinalized();
 
-  modifier onlyFromMigrationMaster() {
-    if (msg.sender != migrationMaster) throw;
-    _;
-  }
-
   function Trustcoin2(address _migrationMaster) {
     if (_migrationMaster == 0) throw;
     migrationMaster = _migrationMaster;
   }
 
   // See ERC20
-  function transfer(address _to, uint _value) returns (bool success) {
+  function transfer(address _to, uint _value) external returns (bool success) {
     balances[msg.sender] = safeSub(balances[msg.sender], _value);
```

The ERC20 standard explicitly mentions a `success` return status, much like ETH's `otherContract.send(value)` . I believe this implementation might not conform to the standard, as it throws when unsuccessful instead of returning `false`. Applies to other methods as well. 
Please let me know if you see this differently.
I know this might make life harder, I apolgoize for that... :face_with_head_bandage:

---

Done